pkgs/shells/nushell/plugins/plot.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+  runCommand,
+  nushell,
+  plot,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "nu_plugin_plot";
+  version = "0-unstable-2025-06-11";
+
+  src = fetchFromGitHub {
+    owner = "Euphrasiologist";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "nu_plugin_plot";
```
Do not rely on pname and repo match